.stylelintrc.js

```diff
@@ -28,7 +28,37 @@ module.exports = {
       // 'float': ['inline-start', 'inline-end', 'none', 'unset'],
       'text-align': ['start', 'end', 'center'],
     },
-    // Disable declaration-strict-value until we are in a position to enforce it.
-    'scale-unlimited/declaration-strict-value': null,
+    // Re-declare declaration-strict-value until we are in a position to enforce it all.
```

Comment:
Can you say more? At least to me it conveys the fact that we still have to override this rule, though now we’re not disabling it, just only configuring it differently.

Reply:
Sorry, Thibaud. I drafted this comment thinking it would become clear to me as I continued reviewing and then I would add a specific suggestion, but never did :)

Now that I've gotten my head around it:

I am assuming that we will always explicitly declare this and are not working toward using it with its stock configuration. I personally would find it a bit obnoxious to have to go to the extremes it demands out-of-the-box, but sorry if I missed some discussion about this.

If my assumption is correct, here's my suggestion:

```suggestion
    // Some parts of declaration-strict-value commented out until we are in a position to enforce them.
```

Reply:
👍 thank you, will apply that change. I’d be interested for your thoughts on which bits of this we _shouldn’t_ apply.

Reply:
You mean my thoughts on why just using the default configuration for `declaration-strict-value` would be obnoxious? [The first example alone on their README](https://github.com/AndyOGo/stylelint-declaration-strict-value#usage) highlights two things I feel pretty sure we want to continue to allow (inherit and currentColor):

> The following patterns are considered warnings:
>
> ```css
> a { color: #FFF; }
> a { color: inherit; }
> a { color: currentColor; }
> ```